src/pages/host/Dashboard.jsx

```diff
@@ -1,3 +1,5 @@
+import { useLoaderData } from "react-router-dom";
```

Comment:
You don't need to import useLoaderData as you are not using it.

Reply:
done